bundle/edu.gemini.sp.vcs/src/main/scala/edu/gemini/sp/vcs/diff/ProgramLocation.scala

```diff
@@ -4,9 +4,39 @@ import scalaz._
 
 sealed trait ProgramLocation
```

Comment:
If you were to add `def fold[A](loc: => A, rem: => A): A = this match ...` you could remove the matches in the `def +(...)` methods below. May or may not be worthwhile.